src/sybil/record_slab.go

```diff
@@ -2,9 +2,9 @@ package sybil
 
 import "time"
 
-func (tb *TableBlock) allocateRecords(loadSpec *LoadSpec, info SavedColumnInfo, loadRecords bool) RecordList {
+func (tb *TableBlock) allocateRecords(flags *FlagDefs, loadSpec *LoadSpec, info SavedColumnInfo, loadRecords bool) RecordList {
```

Comment:
the table doesn't have flags set on it (because a table will eventually be shared between queries) so we need to pass this flags object around everywhere? seems ungainly. one possible solution is a cursor: cursor wraps a table + flags and exposes API, so we don't thread flags around.

Reply:
IMO virtually any flag passing should be turned into better parameters

Reply:
in this case a later commit removes this param (33c69fe4e869319a6c2d356ee2971c02393e6b81 at time of writing)